lib/archethic/self_repair/sync/transaction_handler.ex

```diff
@@ -92,7 +123,10 @@ defmodule Archethic.SelfRepair.Sync.TransactionHandler do
            acceptance_resolver: acceptance_resolver
          ) do
       {:ok, tx = %Transaction{}} ->
-        summary = TransactionSummary.from_transaction(tx)
+        summary =
+          tx
+          |> TransactionSummary.from_transaction(genesis_address)
```

Comment:
I would recommend to add a new flag or create a new function in `TransactionSummary` to select the version of the transaction summary to create